digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/IntegrationApi.scala

```diff
@@ -240,6 +246,33 @@ class IntegrationApi(val massTransitStopService: MassTransitStopService) extends
         .getOrElse("")
   }
 
+  def lastModifiedBy(createdBy: Option[String], modifiedBy: Option[String]): (String, Boolean) = {
+
+    val nonFixedValues = List("split_speedlimit_.*", "batch_process_.*")
+
+    val nonFixedModified = nonFixedValues match {
+      case value if value.exists(nfv => nfv.r.findFirstIn(modifiedBy.getOrElse("")).equals(modifiedBy)) =>
+        modifiedBy
+      case _ => None
+    }
+    val nonFixedCreated = nonFixedValues match {
+      case value if value.exists(nfv => nfv.r.findFirstIn(modifiedBy.getOrElse("")).equals(modifiedBy)) =>
```

Comment:
shouldn't be used the createdBy instead of modifiedBy?  And if so, you can do like this probably : 

```
def lastModifiedBy(...) = {
	val nonFixedModified = obtainFixedValues(modifiedBy)
	val nonFixedCreated = obtainFixedValues(createdBy)
  (...)
}

def obtainFixedValues( str : Option[String] ) {
    val nonFixedValues = List("split_speedlimit_.*", "batch_process_.*")
     nonFixedValues match {
	case value if value.exists(nfv => nfv.r.findFirstIn(str.getOrElse("")).equals(str)) => str
	case _ => None
	}
  }
```